libcrux-ml-dsa/src/simd/portable/vector_type.rs

```diff
@@ -0,0 +1,26 @@
+use crate::simd::traits::COEFFICIENTS_IN_SIMD_UNIT;
+/// Values having this type hold a representative 'x' of the Kyber field.
```

Comment:
This isn't kyber. Probably just c&p but let's fix it.

Reply:
done.